index.js

```diff
@@ -17,7 +17,11 @@ var rsGulp = function (params, callback) {
   }
 
   if (params.output) {
-    formatter = Nsp.formatters[params.output];
+    if (Nsp.formatters.hasOwnProperty(params.output)) {
+      formatter = Nsp.formatters[params.output];
+    } else {
+      return callback('Invalid formatter specified in options. Must be one of ' + Object.keys(Nsp.formatters).join(', ') + '\n');
+    }
```

Comment:
I think this behavior is subtly different than its [**nodesecurity/grunt-nsp** counterpart](https://github.com/nodesecurity/grunt-nsp/blob/e8977c2f50a518c7fe71bdac93a7a58d1ffe7631/tasks/nsp.js#L42-L49):
```js
    if (grunt.option('output')) {
      if (Nsp.formatters.hasOwnProperty(grunt.option('output'))) {
        formatter = Nsp.formatters[grunt.option('output')];
      } else {
        grunt.log.write('Invalid formatter specified in options. Must be one of ' + Object.keys(Nsp.formatters).join(',') + '\n');
      }
    }
```

The Gulp version seems to fail and exit on an invalid formatter, where Grunt warns you that you have an invalid formatter and continues on (using the default formatter).